ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -241,6 +241,9 @@ objects:
         #!/bin/sh
         export _CI_ONLY_STAY_AWAY_OPENSHIFT_INSTALL_AWS_USER_TAGS="{\"expirationDate\": \"$(date -d '4 hours' --iso=minutes --utc)\"}"
         exec /bin/openshift-install --dir=/tmp/artifacts/installer --log-level=debug cluster
+        if [ $? -ne 0 ]; then
+          touch /tmp/shared/exit
+        fi
```

Comment:
We still want this shell script to exit non-zero on failures.  How about we restore the line I removed [here][1] when trying to drop the shell from the `setup` container (which we backed away from in #1761).

[1]: https://github.com/openshift/release/commit/3f2f01c43b51725b685621a95a3acc2115ddfb47#diff-4b206e1e0e8bf659dda6283fbd812433L192

Reply:
While we're at it, we probably want to restore the `TERM` handler too.

Reply:
That sounds more robust... I'm out of time today, so I'll close this PR and either re-open with your fix or leave it close in favor of a replacement from you- whichever comes first. Thanks!